client/plots/cuminc.js

```diff
@@ -1128,9 +1128,7 @@ function setInteractivity(self) {
 		self.app.tip.hide()
 	}
 
-	self.legendClick = function (event) {
-		event.stopPropagation()
-		const d = event.target.__data__
+	self.legendClick = function (d, x, y) {
```

Comment:
Why are `x` and `y` added as parameters to this function? It doesn't look they are being using in this function.